### PR TITLE
Add roadmap to Certify the Collection

### PR DESCRIPTION
**Pull request by john-odonnell**
### Desired Outcome

From ONYX-14258:

> Figure out what we'll need to do to get cyberark/ansible-conjur-collection to Certified level based on our
> [community guidelines](https://github.com/cyberark/community/blob/main/Conjur/conventions/certification-levels.md).

### Implemented Changes

This PR adds a MD document, which investigates the current state of the Collection, and outlines what would be
required to meet Certified project requirements.

### Connected Issue/Story

CyberArk internal issue link: [ONYX-14258](https://ca-il-jira.il.cyber-ark.com:8443/browse/ONYX-14258)

### Definition of Done
*At least 1 todo must be completed in the sections below for the PR to be
merged.*

#### Changelog

- [ ] The CHANGELOG has been updated, or
- [x] This PR does not include user-facing changes and doesn't require a
  CHANGELOG update

#### Test coverage

- [ ] This PR includes new unit and integration tests to go with the code
  changes, or
- [x] The changes in this PR do not require tests

#### Documentation

- [ ] Docs (e.g. `README`s) were updated in this PR
- [ ] A follow-up issue to update official docs has been filed here: [insert issue ID]()
- [x] This PR does not require updating any documentation

#### Behavior

- [ ] This PR changes product behavior and has been reviewed by a PO, or
- [ ] These changes are part of a larger initiative that will be reviewed later, or
- [x] No behavior was changed with this PR

#### Security

- [ ] Security architect has reviewed the changes in this PR,
- [ ] These changes are part of a larger initiative with a separate security review, or
- [x] There are no security aspects to these changes

### CODE REVIEW

File: design/certified_collection_spike.md [hunk 1]
```diff
@@ -0,0 +1,264 @@
+# Ansible Conjur Collection Certified Effort - Spike
```
Comment (rpothier) [inline design/certified_collection_spike.md:1]: Do we need the Spike?

File: design/certified_collection_spike.md [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+
+Figure out what we'll need to do to get
```
Comment (rpothier) [inline design/certified_collection_spike.md:3]: Should it say This document described what we'll ...

File: design/certified_collection_spike.md [hunk 3]
```diff
@@ -0,0 +1,264 @@
...
+[cyberark/ansible-conjur-collection](https://github.com/cyberark/ansible-conjur-collection)
+to Certified level based on our
+[community guidelines](https://github.com/cyberark/community/blob/main/Conjur/conventions/certification-levels.md#certified).
+
+CyberArk internal issue link:
+[ONYX-14258](https://ca-il-jira.il.cyber-ark.com:8443/browse/ONYX-14258)
+
+### What is Ansible?
+
+> Ansible is a radically simple IT automation engine that automates cloud
```
Comment (rpothier) [inline design/certified_collection_spike.md:13]: Are the '>' intended to be there?

**Review by rpothier — COMMENTED** [review]
Looks great! Just a couple minoru comments.

File: design/certified_collection_spike.md (hunk 3 above)
Comment (jtuttle) [reply design/certified_collection_spike.md:13]: Pretty sure they are, these are blockquotes and I think John is quoting lines he has pulled out of Ansible docs / articles.

**Comment by john-odonnell** [comment]
Closing - Collection certified with [v1.2.0 release](https://github.com/cyberark/ansible-conjur-collection/releases/tag/v1.2.0).

**Pull request closed without merging**